src/Pdf/CakePdf.php

```diff
@@ -953,8 +954,12 @@ protected function _render()
             $viewOptions[$var] = $this->{$prop};
         }
 
+        if (!$request = Router::getRequest(true)) {
```

Comment:
Please move the variable assignment above the `if` check.

Reply:
Done 👍